README.md

```diff
@@ -24,10 +24,10 @@
 </p>
 
 
-Turbo is an next-generation toolchain for frontend development, written in Rust. It consists of 3 major parts:
+Turbo is an next-generation, Blazingly Fast™, toolchain for frontend development, written in Rust. It consists of 3 major parts:
```

Comment:
Maybe we can replace ™ with ™️? What do you think?

Reply:
i have never loved that one but i'll sync with Design™ on this

Reply:
Awesome. Let's do that in the next Leadership Sync. 👍

Reply:
Did someone say Leadership Sync?

Reply:
We actually need to take this offline

Reply:
When is the meeting? Do we need to also schedule a follow up?